### PR TITLE
fix(suite): Fix draggable window area on mac

### LINKED ISSUE #13760: Mac - small toolbar area to move app window
**Describe the bug**
It is really hard to move window after latest changes on mac

**Info:**

-   Suite Version: 24.8.0

**Expected behavior**
Manipulating area to be bigger

**Screenshots**

https://github.com/user-attachments/assets/19c1390c-c400-495f-bb7c-ab60fe2ba369

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/13760

## Screenshots:

Red areas are draggable (red is just for highlighting here, it will not be in the app)

image

image

image

image

### CODE REVIEW

File: packages/suite/src/components/suite/TrafficLightOffset.tsx [hunk 1]
```diff
@@ -9,15 +9,23 @@ type Props = {
 };
 
 // See: https://github.com/electron/electron/issues/5678
-const FixForNotBeingAbleToDragWindow = styled.div`
+// Visible all the time in the app
+const ThinFixForNotBeingAbleToDragWindow = styled.div`
     -webkit-app-region: drag;
     height: 12px;
     position: fixed;
+    z-index: 100;
```
Comment (peter-sanderson) [inline packages/suite/src/components/suite/TrafficLightOffset.tsx]: we have some z-index constants? Maybe it shall be used here?

**Review by seibei-iguchi — APPROVED** [review]

**Review by peter-sanderson — APPROVED** [review]
image

**Comment by tomasklim** [comment]
@jvaclavik ship ittttt

**Comment by jvaclavik** [comment]
@tomasklim  it's not ready yet, its still buggy.

**Comment by tomasklim** [comment]
Please fix it also for error case. When error page is available, it is impossible to move the window at all Screenshot 2024-11-07 at 14 34 11

**Comment by tomasklim** [comment]
https://github.com/user-attachments/assets/1644bd8a-cf21-4d33-b0f6-58cbabfb7701

some apps allow to move window also on sides

**Comment by jvaclavik** [comment]
Hope that all issues should be fixed. 

Please can you test it @trezor/qa? 🙏 

Main issues could be: 
- something in header is not clickable/selectable
- window can't be dragged in some cases

**Review by seibei-iguchi — APPROVED** [review]

**Comment by evgenysl** [comment]
Can drag from above green line. Feels good!

Screenshot 2024-11-26 at 12 27 00

Screenshot 2024-11-26 at 12 24 43



Will retest on Intel Mac also.

**Comment by evgenysl** [comment]
Screenshot 2024-11-26 at 13 17 19

**Pull request merged**